package.json

```diff
@@ -1,28 +1,28 @@
 {
   "name": "leaflet.tilelayer.pouchdbcached",
-  "version": "0.3.0",
+  "version": "0.5.0",
   "description": "Allows all TileLayers to cache into PouchDB for offline use",
   "main": "L.TileLayer.PouchDBCached.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/MazeMap/Leaflet.TileLayer.PouchDBCached.git"
+    "url": "git+https://github.com/nikolauskrismer/Leaflet.TileLayer.PouchDBCached.git"
   },
   "keywords": [
     "leaflet",
     "pouchdb"
   ],
-  "author": "Iván Sánchez Ortega <ivan@mazemap.no>",
+  "author": "Nikolaus Krismer <niko@krismer.de>",
   "license": "MIT",
   "bugs": {
-    "url": "https://github.com/MazeMap/Leaflet.TileLayer.PouchDBCached/issues"
+    "url": "https://github.com/nikolauskrismer/Leaflet.TileLayer.PouchDBCached/issues"
```

Comment:
I'd be happy to merge this, but only if the repo would be kept the same

Reply:
reverted this